src/site/layouts/press_release.drupal.liquid

```diff
@@ -0,0 +1,153 @@
+        "fieldPressReleaseFulltext": {
+            "processed": "<p>The new location is just three storefronts away from the current clinic but has more than double</p>\n\n<p>the space at 16,000 square feet.</p>"
+        },
+        "fieldPressReleaseContact": [
+            {
+                "entity": {
+                    "title": "Sheila Tunney",
+                    "fieldDescription": "Public Affairs Specialist",
+                    "fieldPhoneNumber": "412-360-1479",
+                    "fieldEmailAddress": "sheila.tunney@va.gov"
+                }
+            }
+        ],
+        "fieldPressReleaseDownloads": [
+            {
+                "entity": {
+                    "entityId": "17",
+                    "entityBundle": "document",
+                    "name": "VAMC-Region-2019-02-21.pdf",
+                    "fieldDocument": {
+                        "entity": {
+                            "filename": "VAMC-Region-2019-02-21.pdf",
+                            "url": "http://stg.va.agile6.com/sites/default/files/2019-02/VAMC-Region-2019-02-21.pdf"
+                        }
+                    }
+                }
+            },
+            {
+                "entity": {
+                    "entityId": "19",
+                    "entityBundle": "image",
+                    "name": "VHA-Region-Press_Releases-Detail.jpg",
+                    "image": {
+                        "alt": "Veteran's Day poster 2019",
+                        "url": "http://stg.va.agile6.com/sites/default/files/2019-02/VHA-Region-Press_Releases-Detail.jpg"
+                    }
+                }
+            }
+        ],
+        "fieldOffice": {
+            "entity": {
+                "fieldPressReleaseBlurb": {
+                    "processed": "<p><strong>About VA Pittsburgh Healthcare System</strong></p>"
+                }
+            }
+        }
+    },
+{% endcomment %}
+{% include "src/site/includes/header.html" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+            <nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar"></nav>
+            <article class="usa-content">
+                <section class="vads-u-margin-bottom--5">
+                    <h1>{{ title }}</h1>
+                    <p class="vads-u-margin-bottom--1">PRESS RELEASE</p>
+                    <p class="vads-u-font-weight--bold vads-u-margin-bottom--4 vads-u-margin-top--0">{{ fieldReleaseDate.value | formatDate: 'MMMM D, YYYY' }}</p>
+                    <div>
+                        <a href="#" class="vads-u-margin-right--4" onclick="window.print(); return false;"><i class="fa fa-print"></i> Print</a>
```

Comment:
Since there isn't any navigation for print, I think this should be a button, and adding the `usa-button-unstyled` class will remove the button look.

Reply:
ok

Reply:
`usa-button-unstyled` is not working because it is being overwritten by `button, [type=button]` in the css. this is the result: 

browser:

![screen shot 2019-02-26 at 3 30 13 pm](https://user-images.githubusercontent.com/19178435/53454270-81370b00-39db-11e9-8417-193c135fba03.png)

styles:

<img width="335" alt="screen shot 2019-02-26 at 3 30 21 pm" src="https://user-images.githubusercontent.com/19178435/53454271-8300ce80-39db-11e9-8d71-c29b0064d5f6.png">

Reply:
There's a class called `va-button-link` that'll set a button to be styled as a link.

Reply:
Ah that's what it was. Thanks @jbalboni!

Reply:
ok! 'tis updated and looks wunderbar. thank you @jbalboni 😄